linera-views/src/backends/rocks_db.rs

```diff
@@ -402,9 +397,24 @@ impl WritableKeyValueStore for RocksDbStoreInternal {
     const MAX_VALUE_SIZE: usize = MAX_VALUE_SIZE;
 
     async fn write_batch(&self, batch: Batch) -> Result<(), RocksDbStoreInternalError> {
+        let write_root_key = {
+            let mut root_key_written = self
+                .root_key_written
+                .lock()
+                .expect("root_key_written should not be poisoned");
+            if !*root_key_written {
+                *root_key_written = true;
+                true
+            } else {
+                false
+            }
+        };
```

Comment:
```suggestion
        let write_root_key = !root_key_written.fetch_or(true);
```

Reply:
That is great. But the writing is
```
let write_root_key = !self.root_key_written.fetch_or(true, Ordering::SeqCst);
```

which is still shorter than the original code.